config/locales/en.yml

```diff
@@ -263,6 +264,15 @@ en:
       error: 'You cannot leave the meeting, you are the only leader.'
       success: 'You have left %{meeting}'
     new: 'New Meeting'
+    google_cal:
+      create:
+        singular: 'Add to google calendar'
+        success: 'Meeting added to google calendar successfully.'
+        error: 'Something went wrong while adding meeting to the google calendar.'
+      destroy:
+        singular: 'Remove from google calendar'
+        success: 'Meeting removed from google calendar successfully.'
+        error: 'Something went wrong while removing the meeting from the google calendar.'
```

Comment:
Let's change this to 'Something went wrong while removing the meeting from Google Calendar.'